src/index.js

```diff
@@ -133,9 +134,16 @@ function loadLocaleData(locale) {
 const App = ({ locale, messages }) => {
   return (
     <IntlProvider locale={locale} messages={messages}>
-      <Grommet theme={theme} full>
-        <AppBody />
-      </Grommet>
+      <BrowserRouter>
+        <Route exact path='/'>
+          <Grommet theme={theme} full>
```

Comment:
Let' have `<Grommet` component outside the `<Route`.
Please use following hierarchy https://github.com/grommet/grommet-sandbox/blob/master/src/Sandbox.js#L65
```
<IntlProvider
    <Router
        <Grommet
             <Switch
                  <Route
````

Reply:
Should I make the inside of `<Grommet />` Component as a separate component?